README.md

```diff
@@ -54,12 +50,12 @@ Check the [pydantic](https://pydantic-docs.helpmanual.io/usage/schema/) document
 
 > Any config I can change?
 
-Of course. Check the [config](https://spectree.readthedocs.io/en/latest/config.html) document.
+Of course. Check the [config](https://flask-pydantic-spec.readthedocs.io/en/latest/config.html) document.
```

Comment:
are we actually pushing to readthedocs?

Reply:
Not yet, I did a find-replace for this.  I'll get it set up